example/ndpiReader.c

```diff
@@ -884,6 +890,9 @@ static void parseOptions(int argc, char **argv) {
 
     case 'n':
       num_threads = atoi(optarg);
+      if(num_threads > 1)
+        enable_malloc_bins = 0;
```

Comment:
I think a warning to notify the user that memory profiling was deactivated may be helpful. I would go a step further and abort the program to make sure that this parameter configuration does not work.

Reply:
Done